docs/source/algorithms/FFTPreProcessing-v1.rst

```diff
@@ -0,0 +1,90 @@
+.. algorithm::
+
+.. summary::
+
+Allows the user to generate data that has padding (extra measurements of zeros) and/or apply an apodization function.
+
+.. alias::
+
+.. properties::
+
+
+Description
+-----------
+
+This algorithm can prepare data for 
+  :ref:`FFT <algm-FFT>` by applying padding and/or applying an apodization function. 
+
+Padding is when the input data is extended by adding extra measurments of zero at regular intervals after the ned of the input data. 
+
+`Apodization functions <http://mathworld.wolfram.com/ApodizationFunction.html>`_ can be used to remove data with large errors. These are usualy
+found at large time scales. The current implementation includes the following functions:
+
+- None.
+- Bartlett.
+- Lorentz.
+- Connes.
+- Cosine.
+- Gaussian.
+- Welch.
+ 
```

Comment:
I don't think many people know what they are. Math expressions could be used here.
Also it would be helpful if the meaning of `DecayConstant` is explained for each function.